package.json~

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "WebsiteOne-FE",
+  "version": "1.0.0",
+  "description": "Front end for WebsiteOne",
+  "main": "index.js",
+  "repository": "git@github.com:AgileVentures/WebsiteOne-FE.git",
+  "author": "AgileVentures",
+  "license": "MIT",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open",
+    "start-test-server": "webpack-dev-server --mode development --port 8082",
+    "build": "webpack --mode production",
+    "fix": "standard --fix",
+    "test": "export NODE_ENV=testing && standard && jest",
+    "windows-test": "set NODE_ENV=testing&& standard && jest",
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "cypress:ci": "cypress run",
+    "cypress:local": "cypress open --config baseUrl=http://localhost:8082",
+    "cypress:install": "cypress install",
+    "e2e:local": "start-server-and-test start-test-server http://localhost:8082 cypress:local",
+    "e2e:ci": "npm-run-all --parallel --race start:ci 'cypress:ci --config baseUrl=http://localhost:8082'",
+    "start:ci": "webpack-dev-server --config ./webpack.config.js --port 8082 --mode production",
+    "now-build": "yarn build"
+  },
+  "cypress-cucumber-preprocessor": {
+    "nonGlobalStepDefinitions": true
+  },
+  "dependencies": {
+    "axios": "^0.18.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "dotenv-webpack": "^1.7.0",
+    "global": "^4.3.2",
+    "izitoast": "^1.4.0",
+<<<<<<< HEAD
```

Comment:
@nataliyah123... there are several places where you didn't work through the merge conflicts?
a good way to catch these is to run `yarn test` locally cause that will run `standard` and fail if you've forgotten any.

Reply:
i had run the yarn test before i pushed it and i got all cleared test . but when i pushed it though i had done rebasing but it showed merged conflict in two files App.js and initialstate so i fixed them here . but it is still showing error. i will try to fix them as well now